src/components/Editor/Breakpoint.js

```diff
@@ -66,10 +58,17 @@ const Breakpoint = React.createClass({
     this.props.editor.setGutterMarker(line, "breakpoints", null);
     this.props.editor.removeLineClass(line, "line", "new-breakpoint");
     this.props.editor.removeLineClass(line, "line", "has-condition");
-  },
+  }
   render() {
     return null;
-  },
-});
+  }
+}
+
+Breakpoint.propTypes = {
+  breakpoint: PropTypes.object.isRequired,
+  editor: PropTypes.object.isRequired,
+};
+
+Breakpoint.displayName = "Breakpoint";
 
 module.exports = Breakpoint;
```

Comment:
I think we can do `exports default Breakpoint`.

Check other examples for something like this